browser-test/src/support/admin_translations.ts

```diff
@@ -47,23 +60,10 @@ export class AdminTranslations {
     applicationStepDescription = 'step one description',
     blockEligibilityMsg = '',
     programType = 'default',
-    northStar = false,
-  }: {
-    name: string
-    description: string
-    blockName: string
-    blockDescription: string
-    statuses: ProgramStatusTranslationParams[]
-    shortDescription: string
-    applicationStepTitle: string
-    applicationStepDescription: string
-    blockEligibilityMsg?: string
-    programType: string
-    northStar: boolean
-  }) {
+  }: ProgramTranslationParams) {
     await this.page.fill('text=Program name', name)
 
-    if (programType === 'default' || !northStar) {
+    if (programType === 'default') {
```

Comment:
apologies again for not thinking this through before... we don't require long description anymore, even for default type programs. we only require short description. if there's no long description the field will not show in translations. I think we'll want to default long description to an empty string and then only fill the field if it's something other than an empty string, similar to how we do for `blockEligibilityMsg` below.